client/my-sites/earn/home.tsx

```diff
@@ -599,6 +601,7 @@ export default connect(
 			hasSetupAds: Boolean(
 				site?.options?.wordads || isRequestingWordAdsApprovalForSite( state, site )
 			),
+			stripeAvailable: hasConnectedAccount || ! isFreePlan,
```

Comment:
Non-blocking: What do you think about using this opportunity to help the Limited Atomic pod and replace this plan check with a feature check? See p58i-cef-p2

It will require first to add the `memberships` feature to the `WPCOM_Features` class in `wpcom`, and then we could replace this check with something like

```suggestion
			stripeAvailable: hasConnectedAccount || hasActiveSiteFeature( state, siteId, FEATURE_MEMBERSHIPS ),
```

Reply:
> It will require first to add the `memberships` feature to the `WPCOM_Features` class in `wpcom`

Scratch that. After a deeper look, we've realized we should use the `recurring-payments`, `donations` and `premium-content/container` features instead.